ts/features/wallet/component/card/FeaturedCardCarousel.tsx

```diff
@@ -93,6 +157,19 @@ const FeaturedCardCarousel: React.FunctionComponent<Props> = (props: Props) => {
                   />
                 )
               );
+            case ID_CDC_TYPE:
+              return (
+                isCdcEnabled && (
```

Comment:
should the card disappear when a user already has been already enrolled to the CDC bonus? In that case we need to add `isCdcEnrolledSelector` in order to hide the card from carousel.